Removed unused computation of CoG

manumerous opened the pull request:
In the lift drag plugin there were a few lines of code used to compute the center of gravity of the respective link. Since the variables themselves are unused I propose to remove them.

Let me know if the change in the formatting is a problem for your conventions. I can change it if needed.

manumerous committed:
Removed unused computation of CoG
In the lift drag plugin there were a few lines of code used to compute the center of gravity of the respective link. Since the variables themselfes are unused I propose to remove them.

Jaeyoung-Lim reviewed (changes requested):
Can you remove the whitespace fixes? I think this should belong to a different PR

manumerous committed:
Revert "Removed unused computation of CoG"

This reverts commit 97378945e26c16ce3341090c4b453731b2aeadd9.

Reverted changes to undo the formatting.

manumerous committed:
Removed unused computation of CoG
In the lift drag plugin there were a few lines of code used to compute the center of gravity of the respective link. Since the variables themselfes are unused I propose to remove them.

Jaeyoung-Lim reviewed (approved):
Thanks!
